### ISSUE TITLE
Update installation README

### ISSUE THREAD

**Issue by noctivityinc** [hotwired/stimulus-rails#43]
The instructions in the README did not work for Rails 6.1 until I ran this:

```rails webpacker:install:stimulus```

You may want to update.

**Comment by afomera** [comment]
This is fixed in the main branch from GitHub, it was due to a missing index.js file in the controllers folder, and the application.js calling `import "controllers"` after running the install script.  This is the PR where the error was fixed: https://github.com/hotwired/stimulus-rails/pull/34

a comment here https://github.com/hotwired/stimulus-rails/pull/40 references that webpacker:install:stimulus is going to be removed (I guess it's gone in the 6.0 betas for webpack already).

**Comment by noctivityinc** [comment]
I did a clean install today of the master branch and until I ran rails
webpacker:install:stimulus which I got from that comment, it would not
work.

**Comment by afomera** [comment]
I just ran it again on a fresh app too, and don't run into any problem. Can you post what the `rails stimulus:install` script outputs by chance?

Also double check that the gemfile has `gem 'stimulus-rails', github: 'hotwired/stimulus-rails', branch: 'main'` and you've bundled/restarted?

I got this output:

```
± rails stimulus:install
Appending Stimulus setup code to /Users/afomera/Projects/stimulus-test/app/javascript/packs/application.js
      append  app/javascript/packs/application.js
Creating controllers directory
      create  app/javascript/controllers
      create  app/javascript/controllers/hello_controller.js
      create  app/javascript/controllers/index.js
Installing all Stimulus dependencies
         run  yarn add stimulus from "."
yarn add v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > webpack-dev-server@3.11.2" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
warning "webpack-dev-server > webpack-dev-middleware@3.7.3" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 5 new dependencies.
info Direct dependencies
└─ stimulus@2.0.0
info All dependencies
├─ @stimulus/core@2.0.0
├─ @stimulus/multimap@2.0.0
├─ @stimulus/mutation-observers@2.0.0
├─ @stimulus/webpack-helpers@2.0.0
└─ stimulus@2.0.0
✨  Done in 4.43s.
```

Can follow the commits from a initial rails new here: https://github.com/afomera/trying-stimulus-rails/commits/main

I guess the last question if that all fails is _what_ doesn't work? Unable to compile webpack? What kind of error do you see?

**Comment by noctivityinc** [comment]
Well right now it works (except for the double load) but I’ll switch to
that direct branch and see if it still does and maybe solves the other
issue.

Best,

Josh

**Comment by dixpac** [comment]
@noctivityinc , @dhh published latest `0.2.3` with the fix [yesterday](https://github.com/hotwired/stimulus-rails/issues/46) .
So I think we can close this one 😄
